Packages/vcs/vcs/vector.py

```diff
@@ -660,6 +691,8 @@ def list(self):
         print "alignment = ", self.alignment
         print "type = ", self.type
         print "reference = ", self.reference
+        print "scaletype = ", self.scaletype
+        print "scalerange = ", self.scalerange
```

Comment:
good point, let me run all the tests locally here just to make sure. thanks @doutriaux1